lib/renderer/override.js

```diff
@@ -161,11 +161,14 @@ window.open = function(url, frameName, features) {
 // Use the dialog API to implement alert().
 window.alert = function(message, title) {
```

Comment:
Maybe I'm weird but I've used `alert(aBuggyVariable)` for debugging a decent amount.